charmtools/build/__init__.py

```diff
@@ -188,7 +203,7 @@ def status(self):
     def create_repo(self):
         # Generated output will go into this directory
         base = path(self.output_dir)
-        self.repo = (base / self.series)
+        self.repo = (base / self.series if self.series else base)
```

Comment:
Shouldn't it be base / "builds" per https://github.com/juju/charm-tools/issues/115#issuecomment-201084439